### PR TITLE
add more TLS metrics

### PR DESCRIPTION
**Pull request by rphillips**

### CODE REVIEW

File: check/check_http_test.go [hunk 1]
```diff
@@ -292,3 +291,51 @@ func TestHTTPInvalidUrl(t *testing.T) {
 		t.Fatal("should have errored")
 	}
 }
+
+func TestHTTPTLS(t *testing.T) {
```
Comment (itzg) [inline check/check_http_test.go]: If you happen to be in here, adding an underscore before `TLS` would be nice.

File: utils/utils.go [hunk 2]
```diff
@@ -18,3 +18,54 @@
...
 // Most are designed to be usable even outside of the poller project.
 package utils
+
+import (
+	"crypto/x509/pkix"
+	"fmt"
+	"strings"
+)
+
+var oid = map[string]string{
```
Comment (itzg) [inline utils/utils.go]: It would have been nice if the pkix package had declared these already.

File: utils/utils.go [hunk 3]
```diff
@@ -18,3 +18,54 @@
 //
 // Most are designed to be usable even outside of the poller project.
 package utils
+
```
Comment (itzg) [inline utils/utils.go]: Might be nice to put this new content into a file named `certs.go` or similar.

File: check/check_http_test.go (hunk 1 above)
Comment (rphillips) [inline check/check_http_test.go]: fixed in 538526f

File: utils/utils.go (hunk 3 above)
Comment (rphillips) [inline utils/utils.go]: fixed in 0b5a3c0

File: utils/utils.go (hunk 2 above)
Comment (rphillips) [inline utils/utils.go]: Unfortunately, they don't define them in a public way

**Pull request merged**